src/tests/unit/tests/common/components/__snapshots__/header-icon.test.tsx.snap

```diff
@@ -1,17 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`HeaderIconComponent is high contrast mode enabled: false 1`] = `
-<img
-  alt=""
-  className="header-icon"
-  src="../../icons/brand/white/brand-white-48px.png"
-/>
-`;
+exports[`HeaderIconComponent is high contrast mode enabled: false 1`] = `<BrandWhite />`;
```

Comment:
Is there a way to make this more detailed? As in, if I change something inside the svg for `BrandWhite` and dont change the name of this component, this snapshot will still pass. So, this is not much  helpful imo.

Reply:
IMO, that's a feature, not a bug; if we care to defend against regressions in the svg for BrandWhite, that should be in a brand-white.test.tsx, not in a test of its consumer.

Reply:
Not really, using debug yields a very similar result. Also, look at the component and the original test... it was showing a `<img>` tag (with some attributes). The change yield an equivalent result, except the icon components don't have any attribute.